src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -87,11 +87,13 @@ def load_command_table(self, _):
         g.command('get-upgrades', 'get_upgrade_profile', table_transformer=aks_upgrades_table_format)
         g.custom_command('install-cli', 'k8s_install_cli', client_factory=None)
         g.custom_command('list', 'aks_list', table_transformer=aks_list_table_format)
-        g.custom_command('remove-dev-spaces', 'aks_remove_dev_spaces')
+        g.custom_command('remove-dev-spaces', 'aks_remove_dev_spaces',
+                         deprecate_info=g.deprecate(redirect='Bridge To Kubernetes'))
```

Comment:
The `redirect` is used as the new replacement command. It should be a CLI command like `az ...`.

Reply:
I see. We don't have a CLI command to replace it as our team is moving towards a new product. Should I just make this:  `g.custom_command('remove-dev-spaces', 'aks_remove_dev_spaces', deprecate_info=g.deprecate())` ?